UpExtreme/mockup-device-1amp.asl

```diff
@@ -0,0 +1,40 @@
+ /** @file
+  The definition block in ACPI table for RT5682 under SoundWire Controller
+  (link2)
```

Comment:
Maybe edit it to a suitable description.

Reply:
yes, fixed